core-web/libs/sdk/react/src/lib/next/components/DotCMSBodyRenderer/DotCMSBodyRenderer.tsx

```diff
@@ -0,0 +1,38 @@
+import { isInsideEditor } from '@dotcms/client';
+
+import { DotCMSRenderContext } from '../../contexts/DotCMSRenderContext';
+import { DotCMSContentlet, DotCMSPageAsset } from '../../types';
+import { Row } from '../Row/Row';
+
+interface DotCMSBodyRendererProps {
+    dotCMSPageAsset: DotCMSPageAsset;
+    customComponents?: Record<string, React.ComponentType<DotCMSContentlet>>;
+    devMode?: boolean;
+}
+
+export const DotCMSBodyRenderer = ({
+    dotCMSPageAsset,
+    customComponents,
+    devMode
+}: DotCMSBodyRendererProps) => {
+    const dotCMSPageBody = dotCMSPageAsset?.layout?.body;
+    const isDevMode = !!devMode || isInsideEditor();
+
+    if (!dotCMSPageBody) {
+        console.warn('The page body is not defined');
+
+        if (isDevMode) {
+            return <div>The page body is not defined, please provide a valid dotCMSPageAsset</div>;
```

Comment:
Should we style this as we did in the errors POC? We can create a reusable component for it

Reply:
Let's create another ticket for that!

Reply:
I think we have a ticket for the error management, we can tackle this there